### PR TITLE
Included mm/yyyy format

### PR DESCRIPTION
**Pull request by ChiralCarbon**
@kar2905  @anshajgoel Will keep adding formats

### CODE REVIEW

File: Duckling/Time/EN/Rules.hs [hunk 1]
```diff
@@ -683,6 +683,19 @@ ruleHalfHOD = Rule
       _ -> Nothing
   }
 
+ruleMMYYYY :: Rule
```
Comment (anshajgoel) [inline Duckling/Time/EN/Rules.hs]: Will there be other functions for the remaining formats?

Comment (anshajgoel) [inline Duckling/Time/EN/Rules.hs]: And is this detecting the format? Did unit pass?

File: Duckling/Ranking/Classifiers/EN.hs [hunk 2]
```diff
@@ -514,219 +523,223 @@ classifiers
...
+                                    ("absorption of , after named dayFebruary", -4.2017030805426),
                                     ("July<integer> to|till|before <hour-of-day>",
-                                     -5.062595033026967),
-                                    ("tomorrowat <time-of-day>", -5.062595033026967),
-                                    ("tomorrow<time-of-day> am|pm", -5.062595033026967),
-                                    ("after <time-of-day><time-of-day> am|pm", -5.062595033026967),
-                                    ("after <time-of-day>year", -5.062595033026967),
-                                    ("yearminute", -5.062595033026967)],
-                               n = 96}}),
+                                     -5.117993812416755),
```
Comment (anchitbhatia) [inline Duckling/Ranking/Classifiers/EN.hs]: why all the values are changed ?

File: Duckling/Time/EN/Rules.hs (hunk 1 above)
Comment (ChiralCarbon) [inline Duckling/Time/EN/Rules.hs]: Yes

File: Duckling/Ranking/Classifiers/EN.hs (hunk 2 above)
Comment (anchitbhatia) [inline Duckling/Ranking/Classifiers/EN.hs]: can you tell what is it's use?

File: Duckling/Ranking/Classifiers/EN.hs [hunk 3]
```diff
@@ -514,219 +523,223 @@ classifiers
...
-                                    ("monthday", -5.0238805208462765),
-                                    ("monthyear", -3.8199077165203406),
-                                    ("Tuesdaythe <day-of-month> (ordinal)", -5.429345628954441),
-                                    ("Christmasyear", -5.429345628954441),
-                                    ("houryear", -5.429345628954441),
+                                     -4.5217885770490405),
+                                    ("hourday", -3.7333312166847703),
+                                    ("dayhour", -3.1867875103167003),
+                                    ("daymonth", -5.438079308923196),
+                                    ("monthday", -5.032614200815031),
```
Comment (dush1729) [inline Duckling/Ranking/Classifiers/EN.hs]: What these floating value represent? And you have set these by yourself or some reference?

**Pull request merged**